icu4c/source/tools/genrb/parse.cpp

```diff
@@ -1026,6 +1026,15 @@ writeCollationSpecialPrimariesTOML(const char* outputdir, const char* name, cons
         lastPrimaries[i] = static_cast<uint16_t>((data->getLastPrimaryForGroup(UCOL_REORDER_CODE_FIRST + i) + 1) >> 16);
     }
 
+    uint8_t compressibleBytes[32] = {};
+    for (int32_t i = 0; i < 256; ++i) {
+        if (data->compressibleBytes[i]) {
```

Comment:
Note: this field is defined in collationdata.h as

```cpp
    /** 256 flags for which primary-weight lead bytes are compressible. */
    const UBool *compressibleBytes;
```

And that field is initialized in collationdatareader.cpp as

```cpp
    index = IX_COMPRESSIBLE_BYTES_OFFSET;
    offset = getIndex(inIndexes, indexesLength, index);
    length = getIndex(inIndexes, indexesLength, index + 1) - offset;
    if(length >= 256) {
        if(data == nullptr) {
            errorCode = U_INVALID_FORMAT_ERROR;
            return;
        }
        data->compressibleBytes = reinterpret_cast<const UBool *>(inBytes + offset);
    }
```

So it appears to be an array of 256 UBools. (🤞)